src/shadow-embed.js

```diff
@@ -212,7 +227,12 @@ export function importShadowBody(shadowRoot, body, deep) {
     }
   }
   setStyle(resultBody, 'position', 'relative');
-  shadowRoot.appendChild(resultBody);
+  // DO NOT SUBMIT: Find a solution, this is wrong place. Fix the problem above, the style elements need to be placed as the first child
+  //    For example: getRootNode().firstChild , should return a style element
+  const shadowRootElement = doc.createElement('i-amphtml-shadow-root');
+  shadowRootElement.id = shadowRoot.id;
+  shadowRootElement.appendChild(resultBody);
```

Comment:
It looks like this is where the body is duplicated. This code should be simply removed. Not sure where else it's used at this point and why it's here.